### DISCUSSION TITLE
Who is using Chakra UI? ⚡️

### DISCUSSION THREAD

**Discussion by segunadebayo** [chakra-ui/chakra-ui#405] [Show and tell]
Hey Chakra people 😄,

If you or your company is using Chakra UI in production, please leave a comment below so we can mention you in the new README and documentation website.

Here's what your comment should look like:

**Company:** [Put company name] 
**Permission to use our logo in our readme/website:** [Yes or No]
**Project type and size:** [Just a little description of the project & how Chakra helped]
**Project link:** [If public, attach a link]

Thank you

**Comment by franky47** [comment]
**Company**: [47ng](https://github.com/47ng)
**Permission to use our logo in our readme/website**: [Yes](https://res.47ng.com/fse-logo.svg)
**Project type and size**: Personal website/blog, some prototype webapps with Next.js. I initially moved from the Styled Components approach to the style props with a skeptical eye, but in the end it turns out to be awesome. A good base for a style system, with boilerplate-free overrides, it makes UI building fast and fun !
**Project links**: https://francoisbest.com - [`franky47/francoisbest.com`](https://github.com/franky47/francoisbest.com)

Big fan of ChakraUI here, it's usually the first thing I slap onto Next.js ❤️

**Reply by CleoCodeOk** [reply]
Wow

**Comment by KenzoBenzo** [comment]
**Company**: BeFreakingKind apparel
**Permission to use our logo in our readme/website**: [Yes](https://media.graphcms.com/ck7oney0ndq660b209ym6baaa)
**Project type and size**: BFK is an eCommerce site built with Chakra, Stripe, Sendgrid, Printful, GraphCMS, and Netlify. It's a small-sized application that's just getting started. Using Chakra allowed the project to focus on the goal rather than the important nuances of aesthetics and accessibility.
**Project link**: [BeFreakingKind.com](https://BeFreakingKind.com)

---

**Company**: Makenna Smutz Freelancing
**Permission to use our logo in our readme/website:** Yes
**Project type and size**: This is my personal portfolio that I really wanted to play with dark mode.
**Project link**: [Kenna.xyz](https://kenna.xyz)

---

**Company**: [Meeshkan](https://meeshkan.com/)
**Permission to use our logo in our readme/website:** Yes
**Project type and size**: We're a team of 6 people that pivoted to a web app. Chakra allowed us to launch quickly with a high-quality frontend. 
**Project link**: [Meeshkan.com](https://meeshkan.com/)

**Comment by JClackett** [comment]
Company: Common Grounds
Permission to use our logo in our readme/website: Yes
Medium-sized platform for booking co-working spaces worldwide. Chakra allowed us to launch our product super fast! 😃 Next.js, TypeScript, GraphQl, Stripe, Apollo Server, React-native + Expo
Project link: https://www.commongrounds.co/spaces

Chakra has helped so much, now just waiting for React native support so our entire frontend can use Chakra ❤️

**Comment by junaid33** [comment]
Company: Openship
Permission to use our logo in our readme/website: Yes
Project type and size: Opensource.builders allows you to find open-source alternatives to commercial apps. It's used building Chakra-UI and TinaCMS. The code is also open-sourced here: https://github.com/junaid33/opensource.builders.
Project link: https://opensource.builders

Thanks for creating an amazing project!

**Comment by tylerjbainbridge** [comment]
Company: [Cataloged](https://cataloged.co/)
Permission to use our logo in our readme/website: Yes
Project type and size: A knowledge base tool for Links, Files, Notes, Contacts, etc. The company is just me at the moment, but hoping to launch/fundraise soon!
Project link: https://cataloged.co/

![image](https://user-images.githubusercontent.com/13852400/77257978-0972f880-6c4e-11ea-95cd-e5d8867aaaf5.png)

**Comment by bs32g1038** [comment]
**Company: @bs32g1038**
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** Personal website/blog. You can build a similar website by this template quickly. Chakra-UI make our coding more comfortable and fast! 👍 Other libraries such as Next.js, TypeScript, Nestjs, ... also used on Project.

**Project link:** https://www.lizc.net - https://github.com/bs32g1038/node-blog
```js
 I like ChakraUI very much. It’s so excellent! Thank you for your hard work.
```

**Comment by Haja-andri** [comment]
Company: Codelify
Permission to use our logo in our readme/website: Yes
Project type and size: Codelify.dev is a code snippets library for developers to Store, Manage, Retrieve and Share code snippets. Chakra UI helped us focusing on the app logic, while still having beautiful/accessible UI to ship Codelify during a 2 days Hackathons.
Project link: https://codelify.dev

**Comment by breytex** [comment]
**Company:** Need-Mask.com non-profit
**Permission to use our logo in our readme/website:** yes
**Project type and size:** 
We are a team of 6 ppl and created this listing app to connect buyers and suppliers of protective masks and other medical, covid19 relevant gear to fight cutthroat prices. Chakra helped us create this fully-fledged platform in under 10 days. IE11 support is one thing that is missing for us and some breakpoints were glitching on SSR, but everything else was a blast to use. Will use chakra again for my next project!
**Project link:** https://www.need-mask.com/

**Comment by dev-cj** [comment]
Company: (Personal Project by www.devcj.in )
Permission to use our logo in our readme/website: yes
Project type and size: A visualization web app where I used the amazing Chakra UI,
Project description: Visualize corona stats from around the world with downloadable charts to use in projects/research. I will use chakra again for my next project. Thank you so much for your hard work!❤
Project link: https://dev-cj.github.io/visualize-corona/

**Comment by martinpengellyphillips** [comment]
**Company:** Hashbrown
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** A fast capacity planner for team goals. This is a small side hustle to solve a problem we encounter in our jobs, and for which most existing tools seem too light or too heavy 😄. Evaluated a bunch of UI toolkits and Chakra UI was the one that just felt _fun_ to use - the right balance of getting things done, guidance on best practices and the flexibility to adapt as needed. Great community also 🙌 
**Project link:** https://gethashbrown.com

**Comment by remorses** [comment]
Company: Dokz
Permission to use our logo in our readme/website: Yes
Project type and size: Documentation generator built with chakra components!
Project link: https://github.com/remorses/dokz

**Comment by davidli3100** [comment]
**Company**: Donations Revealed
**Permission to use our logo in our readme/website**: yes
**Project type and size**: Corporations have made large anti-racism donations recently - making many people feel as if their donations are insignificant. We made this web app to scale it down to the perspective of an average household to encourage them to make the same relative impact as any given corporation. We currently have over 70 corporations in our database, and will be adding more soon!
**Project link**: [donations.exposed](https://donations.exposed)

**Comment by nfm** [comment]
**Company:** UsabilityHub
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** UsabilityHub is a remote user research platform that takes the guesswork out of design decisions by validating them with real users. We're actively rolling out Chakra in our web app and are very excited for the TS rewrite :tada: 
**Project link:** https://usabilityhub.com

**Comment by feross** [comment]
![social-share](https://user-images.githubusercontent.com/121766/89839268-369f4500-db22-11ea-8b38-ee4f14b68a16.png)

**Company:** Speakeasy
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** I built Speakeasy to help you make a friend in 2 minutes. You're matching over video or voice for a 1-on-1 chat with a random stranger. The conversation lasts 2 minutes (with an "extend" button if the convo is good). My goal is to help people make meaningful connections with strangers. So far we've had 50,000 users try the app.
**Project link:** https://speakeasy.co

Chakra UI has been a joy to use throughout this project!

**Comment by DavidJFelix** [comment]
**Company**: NullServe
**Permission to use our logo in our readme/website**: Yes
**Project type and size**: I've been building NullServe for awhile now and originally it consisted of a Gasby marketing site and CRA site, each using Tailwind CSS for portions of it in different ways. I found Chakra really easy to use within the React ecosystem and I was able to eliminate a large amount of Tailwind-react integration that was present in each app. Chakra also helped ease the pain of creating my own component library so that the look and feel of both the marketing site and application site match.
**Project link**: https://nullserve.com

**Comment by mwood23** [comment]
**Company:** Wave Snippets
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** Wave snippets gives you the ability to create gorgeous animated snippets to share with the world. This is a small product I built to help people learn code through motion.
**Project link:** https://www.wavesnippets.com/

If you'd be open to converting some of the readme to Wave Snippets just say the work and I'll do it!

**Comment by godwinbenson** [comment]
**Company:** Tuteria
**Permission to use our logo:** Yes

**Project type and size:** We re-designed our landing page and onboarding flow for students looking to join our group classes online and offline. Chakra-UI was incredibly helpful in implementing these designs with React very fast, and easily. It was easy to incorporate responsiveness and ensure a consistent layout across all components without having to create my own base component library. We couldn't have pulled this off fast without Chakra-UI. All thanks to the Chakra-UI team, and especially to Segun Adebayo for this amazing work.

**Project link:** https://www.tuteria.com/classes/IELTS/IELTSProgram

**Comment by Sax-Yusuph** [comment]
**Company:** Gitjobs (personal project by [Sax-yusuph](github.com/sax-yusuph) )
**Permission to use our logo:** yes
**Project type and project size:** Gitjobs is a simple job board consuming the github jobs API. It uses react frontend, with an awesome UI using Chakra-ui. It's my first implementation of Chakra-ui, and I was so addicted to it.  Clean components, nice customisation and awesome documentation. Many thanks to the Chakra-ui team

**Project link:** https://gitjobs-2b467.web.app/

**Comment by remjx** [comment]
![image](https://user-images.githubusercontent.com/35121685/91660831-1bcb4b00-eaa6-11ea-87cc-249c41399507.png)

**Company:** Slow TV Map
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** A map of Slow TV videos from YouTube. Components used from Chakra include Modal, Buttons, ColorMode, Radio Buttons, Input, and Toast
**Project link:** https://slowtvmap.com

**Comment by MirzayevFarid** [comment]
![logo-midnight@3x](https://user-images.githubusercontent.com/29518613/91822325-ab106400-ec48-11ea-8bdb-a56ed1ce3ca9.png)

Company: [Radity](https://radity.com/)
Permission to use our logo in our readme/website: Yes
Project type and size:  We develop digital products and build web and mobile applications (20 Team members)

**Comment by gparuthi** [comment]
Company: Arboreum
Permission to use our logo in our readme/website: Yes
Project type and size: A small MVP for a fintech app.
Project link: https://arboreum.dev

**Comment by thattimc** [comment]
![logo-on-light](https://user-images.githubusercontent.com/190269/92305970-8c9ac780-efbe-11ea-9360-8eb7a02c9082.png)

Company: TGAV
Permission to use our logo in our readme/website: Yes
Project type and size: Kickstart is next.js the boilerplate for building apps fast. We using Chakra UI for building up our UI components.
Project link: https://kickstart.app

**Comment by dsumer** [comment]
![logo-mail-template](https://user-images.githubusercontent.com/4418879/93633489-59f5c380-f9ef-11ea-9fc5-81c7474c67d3.png)
**Company:** Project by @anki247 and @dsumer
**Permission to use our logo in our readme/website:** Yes!
**Project type and size:** We've just launched TrueQ, our new developer platform, so we're at the very beginning. We're using Chakra UI to build our UI and we love it as it allows us to develop a high quality product very fast
**Project link:** https://trueq.io

**Comment by paambaati** [comment]
![Logo Full purple](https://user-images.githubusercontent.com/5381764/93910066-321f9c00-fd1e-11ea-87ec-dd332455e0fc.png)

[Media Kit (with SVG logos with our 2 primary brand colours)](https://github.com/chakra-ui/chakra-ui/files/5262795/wiz-logos-2020.zip)

**Company**: Wiz
**Permission to use our logo in our readme/website**: Yes
**Project type and size**: Wiz is a digital freight forwarder based out of Chennai, India. Wiz is a months-old startup aiming to disrupt the freight forwarding space by offering first-class technology-based solutions for freight & logistics. I had a lot of fun building our platform with Chakra-UI, and for someone not very experienced with React, I was able to get quickly productive and launch the initial version of our platform (it had a full booking experience, an admin-only portal for managing customer accounts & their bookings, live-tracking and document management) in only ~3 months!
**Project link**: https://wizfreight.com/

**Comment by MA-Ahmad** [comment]
**Company**: Codenya Studio
**Permission to use our logo in our readme/website**: Yes
**Project type and size**: Personal website to show my work and experience.
**Project link**: https://mahmad.me

Big fan of ChakraUI ❤️

**Comment by sumitngupta** [comment]
**Company**: DoubleLoop
**Permission to use our logo in our readme/website**: [Yes](https://uploads-ssl.webflow.com/5e548d3db0467d0ba8d90988/5e56148f24830b710a09c324_logo-white.png)
**Project type and size**: SaaS product to enable companies to track product iterations and assess the results of them. We recently rebuilt the entire product from scratch and Chakra was instrumental in relaunching fast and actually improve the overall design. 
**Project link**: [doubleloop.app](https://doubleloop.app)

**Comment by maxkostinevich** [comment]
Company: [maxkostinevich](https://maxkostinevich.com/)
Permission to use our logo in our readme/website: Yes
Project type and size: Collection of pre-made UI components to use in admin dashboard.
Project link: [Github Repo](https://github.com/maxkostinevich/Chakra-UI-Admin-Dashboard-Template)

**Comment by locona** [comment]
Company: https://3-shake.com/
Permission to use our logo in our readme/website: Yes
Project type and size: Reckoner is a cloud-based ETL / data pipeline service.
By integrating and linking all kinds of data such as databases, storage applications, etc., we contribute to business transformation utilizing data.
Project link: https://reckoner.io/

**Comment by Prottoy2938** [comment]
Project: Cipher Delta
Permission to use our logo in our readme/website: Yes
Project type: A web application that helps encode and decode plain content. Most of the front-end components used on the application were from Chakra-UI. It also helped integrate a dark mode feature seamlessly. And the discord community was also pretty helpful.
Project link: http://cipher-delta.edrini.xyz/

**Comment by Prottoy2938** [comment]
Project: A Converter Tool
Permission to use our logo in our readme/website: Yes
Project description: An online tool that helps change the color of an SVG using CSS filter 
Project link: https://change-svg-color.edrini.xyz/

**Comment by cereallarceny** [comment]
Company: OpenMined
Permission to use our logo in our readme/website: Yes
Project type and size: Chakra powers our component system OMUI (OpenMined UI), which in turn is used in all of our various web applications. OMUI sits [within a monorepo](https://github.com/OpenMined/openmined) of other websites managed by NX.
Project link: https://openmined.org
About Project: OpenMined is an open-source community whose goal is to make the world more privacy-preserving by lowering the barrier-to-entry to private AI technologies.

![horizontal-primary-light](https://user-images.githubusercontent.com/1297930/99423145-2d2f3100-28f8-11eb-9f43-d4b810267164.png)

**Comment by tolunayakbulut** [comment]
Company: Global CTO Forum

![global-cto-forum](https://user-images.githubusercontent.com/31761883/101293972-60911b80-3825-11eb-9abb-5e7838325523.png)


Permission to use our logo in our readme/website: Yes
Global CTO Forum connects world's leading CTOs and technology stakeholders.
Project link: https://globalctoforum.org/

**Comment by nghiepdev** [comment]
**Company:** 12BAY.VN
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** The airline ticket, train, and bus booking application help you to find cheap flights and trains,... around Vietnam. We using Next.j, ChakraUI, and TailwindCSS to build the product.
**Project link:** [https://12bay.vn](https://12bay.vn/ve-tau)

![image](https://user-images.githubusercontent.com/4768095/103400746-d9f80180-4b78-11eb-8afc-7fdd1989d91e.png)

**Comment by waptik** [comment]
Company: Project by [Stephane](https://twitter.com/_waptik)
Permission to use our logo in our readme/website: Yes
Project type and size: Ku-di is an alternative to Patreon, Ko-fi, Buymeacoffee for Africa. Only Ghanaian users can receive donations at the moment, more African countries to be added soon.
Project link: [Ku-di](https://ku-di.com)

**Comment by Prottoy2938** [comment]
Company: SurveyWave
Permission to use our logo in our readme/website: Yes
Project type and size: Most of its design components are from Chakra-UI. It's prop like component style helped fasten the staying process.
Project link: http://surveywave.xyz/

![app-demo](https://user-images.githubusercontent.com/51432728/104315369-0ca0e300-5505-11eb-9b85-c98a3b8495c2.gif)

**Comment by kodgurkini** [comment]
![tabExtendWebLogo@2x](https://user-images.githubusercontent.com/20929184/105083666-856be600-5a95-11eb-97db-642e8e6d5dcb.png)
Company: tabExtend
Permission to use our logo in our readme/website: Yes
Project type and size: Built a browser extension(Chrome, FF and Edge so far) that replaces the new tab with buckets(spaces) where the users can drop of tabs, write notes and create to-dos. All of the UI is built with Chakra UI as a foundation. I could honestly not have built this without Charka so a big shoutout to you guys! 
Project link: https://www.tabextend.com

**Comment by remjx** [comment]
![Coindrop logo](https://coindrop.to/images/logo-light.png)
**Company:** Coindrop
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** Accept zero-fee donations and tips anywhere. Created by me.
**Project link:** https://coindrop.to

**Comment by stellarhoof** [comment]
**Company**: SmartProcure
**Permission to use our logo in our readme/website**: Yes
**Project type and size**: Government procurement space. We've been slowly transitioning our frontend to use Chakra UI. Our site is fairly large, with 1200+ files
**Project link**: https://smartprocure.us

Logo: https://app.govspend.com/static/media/logo-full.23d2bb14.svg

**Comment by malhotrachetan** [comment]
Company: Narration Box
Permission to use our logo in our readme/website: Yes
Project type and size: Early stage Saas startup in the voice and audio space. MIgrated to Chakra components from tailwind. 
Project link: https://narrationbox.com/

Logo: https://narrationbox.com/static/97986fc8d15fff13914c2a0307646574/73c85/logo.png

**Comment by darrendube** [comment]
**Company:** NeoCapitalist
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** A blog that simplifies the complex Economic machine - for Economics students and casual readers alike. The site's design is built with Chakra UI.
**Project link:** https://neocapitalist.darrendube.com/

**Comment by VamiqValji** [comment]
![FeatureGraphicLinkedin](https://user-images.githubusercontent.com/71946460/130257274-4b81f845-0210-4fc4-847d-9ad2848224e2.png)
Company: Fitprovement
Permission to use our logo in our readme/website: Yes
Project type and size: It's the social media for workouts. Share your own workouts, try others', and become more fit offline and on the go!  Chakra UI allowed me to fiddle less with UI styling, and focus more on the logic side of the code. This sped up my development process and also helped me learn about design patterns just through viewing the components you guys had put together. Chakra UI is amazing! 🎉
Project link website: https://www.fitprovement.com/
Project link mobile app: https://play.google.com/store/apps/details?id=com.vamdev.fitprovement

**Comment by rossng** [comment]
![image](https://user-images.githubusercontent.com/565371/132094434-2366b872-c74a-4d02-9a3f-59296f9200af.png)

Company: Midspace
Permission to use our logo in our readme/website: Yes
Project type and size: 

[Midspace](https://midspace.app/) is an open-source platform for running accessible, climate-friendly virtual and hybrid conferences. In particular, we're focused on academic conferences and research conferences. Midspace provides some sophisticated tools for running large multi-track academic conferences. For example: 

- Midspace can automatically play prerecorded videos mixed with live content.
- Organisers can publish detailed information about their proceedings.
- Midspace generates automated captions for videos that can be corrected by their authors.
- Conference attendees can socialise through chat, randomised 'shuffle' networking rooms and a bunch of other interactive features.

We host a cloud version of Midspace to make it easy for conference organisers to get started. But you can also run your own instance from the open-source code with no restrictions - we don't hide proprietary features behind a paywall.

We've found Chakra UI really pleasant to work with over the last year or so. And it dovetails nicely with our aim of making conferences more inclusive and accessible! I look forward to seeing where it goes in the future.

Project link: https://midspace.app/, https://github.com/clowdr-app/clowdr/

**Comment by declanramsay** [comment]
**Company:** International Quidditch Association
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** The International Quidditch Association is the international governing body for the sport of quidditch. Quidditch is played in over 30 countries, and the International Quidditch Association facilitates competition and cooperation between our member National Governing Bodies through resources and competitions. This includes providing rulebooks for the sport, and running events like World Cup, Pan American Games, and European Games.

The website was recently rebuilt from a Wordpress + Template site to a Next.js and Headless CMS, with Chakra as our primary component and theme library.

**Project link:** https://iqasport.org

![Screenshot 2021-10-17 at 17-54-02 International Quidditch Association](https://user-images.githubusercontent.com/10368230/137637212-b66f545f-df8c-478f-9255-8aee2b5dc222.png)

**Comment by DonaldLouch** [comment]
**Company**: DevLexicon
**Permission to use our logo in our readme/website**: [Yes](https://devlexicon.vercel.app/devLexiconTitle.svg) (UPDATED LINK)
**Project type and size**: 

> We're using Chakra UI to extend our theme's capabilities to allow for a more rich user experience. We re currently re-modelling and re-coding our portal website from a PHP/HTML/CSS web stack to a NextJS/Chakra/Firebase web stack.

**Project link**: [beta.devlexicon.ca](https://beta.devlexicon.ca) (NOTE that the website is still be constructed)

<img width="1680" alt="Screen Shot 2021-10-19 at 7 38 52 PM" src="https://user-images.githubusercontent.com/16690527/138019077-8fd59c27-37dc-41d5-9a13-fa3a65b25856.png">

**Comment by MA-Ahmad** [comment]
**Company: TemplatesKart**
**Permission to use our logo in our readme/website: Yes**

[TemplatesKart](https://templateskart.com) is a set of accessible and reusable components that are commonly used in web applications. It provides production ready hand crafted Projects and Components list built on top of Chakra UI.

**Project link: https://templateskart.com**
**Project Repo: https://github.com/MA-Ahmad/templateskart**

![TemplatesKart](https://user-images.githubusercontent.com/37842853/166335520-807ce391-c69a-47a9-85a0-d26c3830f7ee.png)

**Comment by hosmelq** [comment]
**Company:** PickupHub
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** Project that allows sports facilities administrator to sell positions for games.
Project link: https://pickuphub.net (You need to sign in since the landing is a WordPress)

![pickuphub net_hub](https://user-images.githubusercontent.com/1166143/194991568-866a2d33-120b-4501-89f8-d549ef2367f1.png)

**Comment by amosbastian** [comment]
Company: MentionFunnel
Permission to use our logo in our readme/website: Yes
Project type and size: MentionFunnel is a social listening tool that will help you easily find leads who mention your keywords on social media and let you engage with them directly
Project link: https://mentionfunnel.com/

![image](https://user-images.githubusercontent.com/9199433/208177363-70b1620c-0e4c-430e-8eb3-9f8086542f11.png)
![image](https://user-images.githubusercontent.com/9199433/208177438-b1dfda31-4e2e-448f-b8ae-dbc74b920e69.png)

**Comment by ankursehdev** [comment]
Company: Loblaws Inc. 
Permission to use our logo in our readme/website: Yes

Project type and size: 
1. Loblaws Inc. is one of the biggest Canadian supermarket chains with over 250,000 employees and 14 different lines of business. We are creating a new component library using ChakraUI and it has been so easy to customize it and embed it into our design system.
2. The idea of creating your own design system is what was most attractive and it has helped up to quickly develop so many parts of the website.

Project link: [Loblaws](https://www.loblaws.ca/)

**Comment by aleksbobic** [comment]
Company: Collaboration Spotting X
Permission to use our logo in our readme/website: Yes
Project type and size: Collaboration Spotting X is a network based visual analytics and data retrieval tool which I developed as part of my doctoral studies. It enables users to load custom datasets in .CSV format and analyse them using various network analysis techniques as well as using multiple alternative data representations in the form of widgets.
Project link: [Collaboration Spotting X Github page](https://github.com/aleksbobic/csx)

![Collaboration Spotting X - Preview](https://github.com/aleksbobic/csx/blob/master/cover.gif?raw=true)

**Comment by bfeister** [comment]
Company: Salesforce (and many customer companies)
Permission to use our logo in our readme/website: No (I can't grant this, but legal could be approached if desired, reach out to me)
Project type and size: Ecommerce site used as a backing for a number of very large retailers. Including https://buy.jaguar.co.za/ and others
Project link: [Salesforce Composable Commerce Demo Store](https://pwa-kit.mobify-storefront.com/)

<img width="1792" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/47546998/ae2f6b5b-99a0-4d95-a9bf-34481c614244">

<img width="1792" alt="image" src="https://github.com/chakra-ui/chakra-ui/assets/47546998/919c1f18-d2b0-48d0-8e7c-c41ae0eff020">

**Comment by wintkhantlin** [comment]
Company: Happer64Bit
Permission to use our logo in our readme/website: Yes
Project type and size: Blog and Homepage
Project link: https://happer64bit.vercel.app

**Comment by ghost** [comment]
Company: Netspot Solutions
Permission to use our logo in our readme/website: Yes
Project Type: commercial website/landing page
Project Link: [netspot.solutions](https://netspot.solutions/)
![image](https://github.com/chakra-ui/chakra-ui/assets/94535263/c46a305e-db2a-4c09-bb13-3c2199a65830)

**Comment by mahtab-quixgo** [comment]
Company : Quixgo Infotech
Permission to use our logo in our readme/website: Yes
Project type and size: Using Next Js, for SEO website , www.quixgo.com, and using React Js for all Internal Application.
Project Link: [https://quixgo.com/](https://quixgo.com/)
![image](https://github.com/user-attachments/assets/b1279e56-c516-4764-bd26-9ffd3b1175cf)

**Comment by techwithtwin** [comment]
Company: TECHWITHTWIN
Permission to use our logo in our readme/website: yes
Project type and size: Revamped our company website.
Project link: https://techwithtwin.com

**Comment by anki247** [comment]
Company: JetCoders GmbH
Permission to use our logo in our readme/website: YES
Project type and size: Company website. We love Chakra-UI. It is the default lib we use for all of our projects. 🚀
Project link: [JetCoders.at](https://www.jetcoders.at)

**Comment by vitalijalbu** [comment]
https://resthotels.it
abbandoned, v2 (3 yrs ago)

**Comment by dsumer** [comment]
Company: Vemetric
Permission to use our logo in our readme/website: Yes
Project type and size: Simple yet powerful analytics platform. Wouldn't want to build the UI with anything else than Chakra.
Project link: https://vemetric.com

**Comment by anki247** [comment]
Company: JetCoders GmbH
Permission to use our logo in our readme/website: Yes
Project type and size: An advanced CMS solution for mid size companies. It is build for speed and performance and Charka V3 delivers 💪
Project link: [jet-cms.com](https://jet-cms.com/en)

**Comment by TheLostByGass** [comment]
**Company**: revoBill 
**Permission to use our logo in our readme/website:** Yes
**Project type and size:** 
revoBill is a SaaS invoicing tool for businesses (primarily focused on Germany and Austria) with a focus on automating flows between quotes to invoices to payments. It is built entirely with Chakra UI v3. It powers our landing page and our full application including onboarding, quoting, invoicing, customer management, product catalog, approval flows and Stripe Connect integration. Chakra helped us ship a polished, accessible UI very quickly without the need of heavy design knowledge - and maintenance is super lightweight.
**Project link:** Landing page: https://www.revobill.de/  | App: https://app.revobill.de/ 

Huge fan of Chakra! Keep it going 🚀  My first thing to import when scaffolding a new project 🥇
